modules/virt-temporary-token-VNC.adoc

```diff
@@ -67,3 +68,15 @@ $ oc login --token ${VNC_TOKEN}
 ----
 $ virtctl vnc <vm_name> -n <namespace>
 ----
+
+.Known issue
+
+* Revoking a specific token is not currently possible.
++
+If you require revoking a token, you must delete the service account that was used for creating it. However, this also revokes all other tokens that were created by using the service account.
++
+[source,terminal]
+----
+$ virtctl delete serviceaccount --namespace "<namespace>" "<vm_name>-vnc-access" 
```

Comment:
Not sure if virtctl is different than oc, but for oc we normally don't use quotes around arguments

```suggestion
$ virtctl delete serviceaccount --namespace <namespace> <vm_name>-vnc-access
```

Fine to leave as is though if that's right for virtctl

Reply:
As per kubevirt docs, the quotation marks should be there - https://github.com/kubevirt/vm-console-proxy/blob/main/docs/api.md#revoking-a-token (possibly to make sure that the values still work if the replaceables are replaced by more complex strings).